src/main/java/core/basesyntax/FileWork.java

```diff
@@ -8,7 +14,30 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
+
+    public static final char CONSTW = 'w';
+
     public String[] readFromFile(String fileName) {
-        return null;
+        ArrayList<String> wordsWithWList = new ArrayList<String>();
+        String text = "";
+        try {
+            text = new String(Files.readAllBytes((Paths.get(fileName)))).toLowerCase().replaceAll(
```

Comment:
let's use readAllLines() method. This way you will get List<String> and won't have to use split("\n") to divide text to lines